[2021-05-17 09:31:45,442] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: matches_links.get_autumn_nations_cup_links 2021-04-24T10:00:00+00:00 [queued]>
[2021-05-17 09:31:45,446] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: matches_links.get_autumn_nations_cup_links 2021-04-24T10:00:00+00:00 [queued]>
[2021-05-17 09:31:45,447] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-17 09:31:45,447] {taskinstance.py:1069} INFO - Starting attempt 5 of 6
[2021-05-17 09:31:45,447] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-17 09:31:45,451] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): get_autumn_nations_cup_links> on 2021-04-24T10:00:00+00:00
[2021-05-17 09:31:45,453] {standard_task_runner.py:52} INFO - Started process 24711 to run task
[2021-05-17 09:31:45,460] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'matches_links', 'get_autumn_nations_cup_links', '2021-04-24T10:00:00+00:00', '--job-id', '10', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/matches_links/matches_links.py', '--cfg-path', '/var/folders/tp/ft4rfkmn1kz3dyw8hlfz1dxw0000gn/T/tmp_yqynsev', '--error-file', '/var/folders/tp/ft4rfkmn1kz3dyw8hlfz1dxw0000gn/T/tmp8lx39i9n']
[2021-05-17 09:31:45,461] {standard_task_runner.py:77} INFO - Job 10: Subtask get_autumn_nations_cup_links
[2021-05-17 09:31:45,488] {logging_mixin.py:104} INFO - Running <TaskInstance: matches_links.get_autumn_nations_cup_links 2021-04-24T10:00:00+00:00 [running]> on host Henrys-MacBook-Pro.local
[2021-05-17 09:31:45,510] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=matches_links
AIRFLOW_CTX_TASK_ID=get_autumn_nations_cup_links
AIRFLOW_CTX_EXECUTION_DATE=2021-04-24T10:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=backfill__2021-04-24T10:00:00+00:00
[2021-05-17 09:31:45,510] {bash.py:135} INFO - Tmp dir root location: 
 /var/folders/tp/ft4rfkmn1kz3dyw8hlfz1dxw0000gn/T
[2021-05-17 09:31:45,511] {bash.py:158} INFO - Running command: Rscript /Users/harry/rugby_data_project_airflow/R/scripts/create_matches_table.R  https://www.rugbypass.com/autumn-nations-cup/matches/ 2021-04-24
[2021-05-17 09:31:45,517] {bash.py:169} INFO - Output:
[2021-05-17 09:31:45,687] {bash.py:173} INFO - Error in exc_date %>% dmy() : could not find function "%>%"
[2021-05-17 09:31:45,687] {bash.py:173} INFO - Execution halted
[2021-05-17 09:31:45,698] {bash.py:177} INFO - Command exited with return code 1
[2021-05-17 09:31:45,704] {taskinstance.py:1482} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/harry/.local/share/virtualenvs/rugby_data_project_airflow-d2RAFQI6/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1138, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/Users/harry/.local/share/virtualenvs/rugby_data_project_airflow-d2RAFQI6/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1311, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/Users/harry/.local/share/virtualenvs/rugby_data_project_airflow-d2RAFQI6/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1341, in _execute_task
    result = task_copy.execute(context=context)
  File "/Users/harry/.local/share/virtualenvs/rugby_data_project_airflow-d2RAFQI6/lib/python3.8/site-packages/airflow/operators/bash.py", line 180, in execute
    raise AirflowException('Bash command failed. The command returned a non-zero exit code.')
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code.
[2021-05-17 09:31:45,708] {taskinstance.py:1525} INFO - Marking task as UP_FOR_RETRY. dag_id=matches_links, task_id=get_autumn_nations_cup_links, execution_date=20210424T100000, start_date=20210517T083145, end_date=20210517T083145
[2021-05-17 09:31:45,726] {local_task_job.py:146} INFO - Task exited with return code 1
